mmv1/products/cloudbuildv2/api.yaml

```
@@ -0,0 +1,42 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: CloudBuildv2
```

```suggestion
name: Cloudbuildv2
```

This aligns the intended product naming w/ DCL naming, where `cloudbuildv2` is the slug (while this currently generates `cloud_buildv2`). This change should also remove the need for the `legacy_name` in `terraform.yaml`.